db/seeds.rb

```diff
@@ -29,18 +29,18 @@ def create_stickers
                                          image: "http://devstickers.com/assets/img/cat/react-js.png",
                                          price: 8,
                                          description: "React.js logo")
+    @angularjs_sticker = @javascript_category.stickers.create(title: "Angularjs",
+                                         image: "http://devstickers.com/assets/img/cat/angularjs.png",
```

Comment:
Line is too long. [102/80]<br>Align the elements of a hash literal if they span more than one line.